<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ZComposeVideoDlgClass</class>
 <widget class="QDialog" name="ZComposeVideoDlgClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>314</width>
    <height>164</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ZComposeVideoDlgClass</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <property name="verticalSpacing">
      <number>10</number>
     </property>
     <item row="0" column="1">
      <widget class="QLineEdit" name="fps"/>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="filename">
       <property name="placeholderText">
        <string>(Option)</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="filenamelbl">
       <property name="text">
        <string>File Name:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="pathlbl">
       <property name="text">
        <string>Path:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLineEdit" name="linePath">
         <property name="placeholderText">
          <string>(Option)</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnOpen">
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="fpslal">
       <property name="text">
        <string>FPS:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="bitrate"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="bitratelgl">
       <property name="text">
        <string>Bit Rate:</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="btnGroup">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
